code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2173,3 +2173,24 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/himalayan
+	name = "Himalayan"
+	description = "Freezing cold; its a miracle"
+	boozepwr = 40
+	color = "##0000ff"
```

Comment:
one # too many, although it could still be functional

Reply:
Must've not deleted the hastag when copy pasting the new color in, and I don't think how it looked in a syringe or beaker is something I tested in game. Oops!